content/topics/strategy/power-bi/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+content_type: topic
+ready: true
+title: Power BI
+---
+
+## Power BI
```

Comment:
Please remove the heading as one is auto generated from the title

Reply:
@chrisgoutzi Hey, can you please polish this off? You're rreally close to being complete.

When hugo renders this thing then the html will ook like this:

```
<h1>Power BI</h1>
<h2>Power BI</h2>
```
As you can see this is redundant and also not dry and also redundant.

Please run the application and see for yourself. The best thing to do is remove the line that says `## Power BI`

@asradebe can you please follow upi on this?

Reply:
will do

Reply:
@asradebe  should we close this PR? It looks like the contributor isn't interested in finishing